cmd/zpool/zpool_main.c

```diff
@@ -506,7 +506,8 @@ get_usage(zpool_help_t idx)
 		return (gettext("\tinitialize [-c | -s | -u] [-w] <pool> "
 		    "[<device> ...]\n"));
 	case HELP_SCRUB:
-		return (gettext("\tscrub [-s | -p] [-w] [-e] <pool> ...\n"));
+		return (gettext("\tscrub [-s | -p] [-w] [-e] [-C] [-R] "
```

Comment:
```suggestion
		return (gettext("\tscrub [-e | -p | -s | -C | -R] [-w] "
```
They all seem to be mutually exclusive.

Reply:
Thx, done!